hummingbot/strategy/spot_perpetual_arbitrage/arb_proposal.py

```diff
@@ -13,23 +14,25 @@ class ArbProposalSide:
     def __init__(self,
                  market_info: MarketTradingPairTuple,
                  is_buy: bool,
-                 order_price: Decimal
+                 order_levels: List[List[Decimal, Decimal]]
                  ):
         """
         :param market_info: The market where to submit the order
         :param is_buy: True if buy order
-        :param order_price: The price required for order submission, this could differ from the quote price
+        :param order_levels: A list of tuples, each tuple contains (order_price, order_amount)
```

Comment:
ArbProposalSide will not hold the orderbook levels for each leg up to max_possible_arbitrage_amount.